RUNNING_DOCKER.RST

```diff
@@ -0,0 +1,369 @@
+Development Setup
+=================
+
+Cloning the project
+-------------------
+
+::
+
+    git clone git@github.com:SFDO-Tooling/MetaCI
+    cd MetaCI
+
+Docker and Docker-Compose Installation:
+--------------------------------------
+To download and install docker please visit: https://hub.docker.com/?overlay=onboarding 
+and follow the installation instructions to download docker if needed.
+To verify you have successfully installed docker type:
+
+::
+    
+    docker -v
+
+You should see something like the following:
+
+``Docker version 19.03.4, build 9013bf5``
+
+To download and install docker-compose please visit: https://docs.docker.com/v17.09/compose/install/
+and follow the installation instructions to download docker-compose if needed.
+To verify you have successfully installed docker-compose type:
+
+::
+
+    docker-compose -v
+
+You should see something like the following):
+
+``docker-compose version 1.16.1, build 6d1ac219``
+
+Running MetaCI In Docker:
+=========================
+
+To run MetaCI on Docker there are 3 major steps necessary:
+----------------------------------------------------------
+
+- `.env File Creation and Variable Declaration`_
+.. _.env File Creation and Variable Declaration: https://github.com/SFDO-Tooling/MetaCI/blob/feature/docker/CONTRIBUTING.rst#env-file-creation-and-variable-declaration
+
+- `Building Your Docker Containers`_
+.. _Building Your Docker Containers: https://github.com/SFDO-Tooling/MetaCI/blob/feature/docker/CONTRIBUTING.rst#building-your-docker-containers
+
+- `Running Your Docker Containers`_
+.. _Running Your Docker Containers: https://github.com/SFDO-Tooling/MetaCI/blob/feature/docker/CONTRIBUTING.rst#running-your-docker-containers
+
+
+.env File Creation and Variable Declaration
+-------------------------------------------
+
+*Please begin by making a copy of env.example and renaming it .env in your root project directory*
+
+Variables Required Locally
+---------------------------------
+
+DJANGO_SECRET_KEY: 
+    This represents the secret key for the django web application, for local testing, 
+    arbritary strings such as the one given in the env.example will suffice. 
+    Otherwise use your production secret key.
```

Comment:
It's an arbitrary string for production too. I think it's used to sign session cookies; the important thing is that it's not copied from another Django site.